ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/HikariJdbcConnectionProvider.scala

```diff
@@ -75,6 +79,8 @@ class HikariJdbcConnectionProvider(dataSource: HikariDataSource, healthPoller: T
       Unhealthy
 
   override def runSQL[T](block: Connection => T): T = {
+    //note that Hikari uses auto-commit by default.
+    //in `runSql` below, the `.close()` will automatically trigger a commit.
```

Comment:
Well, it won't, because we disable auto-commit two lines down.

Reply:
heh…should I just drop this comment then? It looked like it was in the wrong place, but now it just seems completely out of date.

Reply:
Yeah, I think so. I think this changed a few months ago.